src/test/java/StorageTest.java

```diff
@@ -0,0 +1,28 @@
+import org.junit.Test;
+
+import java.io.File;
+import java.util.ArrayList;
+
+import static org.junit.Assert.assertEquals;
+
+public class StorageTest {
+    @Test
+    public void emptyFileTest() {
```

Comment:
You can add more description to the name using three part format. (Under Naming in coding standard)